src/main/java/com/ericsson/ei/subscriptionhandler/RunSubscription.java

```diff
@@ -64,19 +65,10 @@ public boolean runSubscriptionOnObject(String aggregatedObject, Iterator<JsonNod
         boolean conditionFulfilled = false;
         int count_condition_fulfillment = 0;
         int count_conditions = 0;
-
         int requirementIndex = 0;
 
         while (requirementIterator.hasNext()) {
 
-            JsonNode aggrObjJsonNode = null;
-            ObjectMapper objectMapper = new ObjectMapper();
-            try {
-                aggrObjJsonNode = objectMapper.readValue(aggregatedObject, JsonNode.class);
-            } catch (Exception e) {
-                LOGGER.error(e.getMessage(), e);
-            }
-
```

Comment:
"aggrObjJsonNode" object was never used in the code below, so removed this unused code section.